server/.env.example

```
@@ -15,4 +15,9 @@ CLDN_API_KEY=[api_key_cloudinary_aqui]
 CLDN_SECRET_KEY=[secret_key_cloudinary_aqui]
 STRIPE_SECRET_KEY=[clave_secreta_stripe_aqui]
 STRIPE_WEBHOOK_SECRET=[clave_webhook_stripe_aqui]
-ML_SERVICE_URL=[url_del_microservicio_ml_aqui]
\ No newline at end of file
+ML_SERVICE_URL=[url_del_microservicio_ml_aqui]
+RESERVATION_SCORE_ALPHA=0.4    # Peso de posición en cola
+RESERVATION_SCORE_BETA=0.3     # Peso de puntualidad histórica
+RESERVATION_SCORE_GAMMA=0.2    # Peso de tiempo de espera
+RESERVATION_SCORE_DELTA=0.1    # Penalización por devoluciones tardías
+LOAN_FEE_ESTIMATE=5.00  # Fee estimado por préstamo para optimización del carrito
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)